pkg/ui/app/containers/nodeGraphs.tsx

```diff
@@ -446,15 +521,26 @@ class NodeGraphs extends React.Component<NodeGraphsProps, {}> {
           <SummaryStat title="Total Nodes" value={this.props.nodeCount} />
           <SummaryStat title="Capacity Used" value={capacityPercent}
                        format={(v) => `${d3.format(".2f")(v)}%`}
-                       tooltip={`You are using ${Bytes(capacityUsed)} of ${Bytes(capacityTotal)} 
+                       tooltip={`You are using ${Bytes(capacityUsed)} of ${Bytes(capacityTotal)}
                        storage capacity across all nodes.`} />
+          <SummaryStat title="Unavailable ranges" value={this.props.unavailableRanges} />
+          <SummaryStat title="Queries per second" value={this.props.queriesPerSecond} />
+          <SummaryStat title="P50 latency" value={this.props.p50Latency} format={(n) => d3.format(".1f")(NanoToMilli(n)) + " ms"} />
```

Comment:
I considered that approach but this seemed much more straightforward, though I'll agree that the logic probably shouldn't all be in `nodeGraphs.tsx`. 

I examined it further and the most difficult component to mess with is not `MetricsDataProvider` but rather the `metrics` state. I'd have to change the way `nextRequest` is used. It honestly doesn't seem worth it to modify this logic especially since we know we'll only ever have one set of current data that we want for each page. We can just use the `cachedDataReducer` to make and store the current data request. The question is where it should live.